src/gui/widgets/Knob.cpp

```diff
@@ -497,8 +509,9 @@ float Knob::getValue( const QPoint & _p )
 {
 	float value;
 
-	// arcane mathemagicks for calculating knob movement
-	value = ( ( _p.y() + _p.y() * qMin( qAbs( _p.y() / 2.5f ), 6.0f ) ) ) / 12.0f;
+	// knob value increase is linear to mouse movement
+	// for larger screens, change the knob slower
+	value = .09f * (_p.y()*(1080/s_desktopSize.y()));
 
```

Comment:
` (_p.y()*(1080/s_desktopSize.y()))` These are all int values right?
Maybe force floating point computation to avoid integer math with truncation?
`value = .09f * (_p.y()*(1080.f/s_desktopSize.y()));`

Reply:
Oh how could I not have seen this. Of course, on my screen with 1080 height this always worked, but that explains a lot.

I pushed a fix now, sorry for that.